packages/osl-operator-bundle-image/generated/bundle/manifests/logic-operator-rhel8-controllers-config_v1_configmap.yaml

```diff
@@ -28,36 +28,35 @@ data:
+    jobsServicePostgreSQLImageTag: "docker.io/apache/incubator-kie-kogito-jobs-service-postgresql:1.36.0"
+    jobsServiceEphemeralImageTag: "docker.io/apache/incubator-kie-kogito-jobs-service-ephemeral:1.36.0"
     # The Data Index image to use, if empty the operator will use the default Apache Community one based on the current operator's version
-    dataIndexPostgreSQLImageTag: "docker.io/apache/incubator-kie-kogito-data-index-postgresql:main"
-    dataIndexEphemeralImageTag: "docker.io/apache/incubator-kie-kogito-data-index-ephemeral:main"
+    dataIndexPostgreSQLImageTag: "docker.io/apache/incubator-kie-kogito-data-index-postgresql:1.36.0"
+    dataIndexEphemeralImageTag: "docker.io/apache/incubator-kie-kogito-data-index-ephemeral:1.36.0"
+    # The Kogito PostgreSQL DB Migrator image to use (TBD: to replace with apache image)
+    dbMigratorToolImageTag: "docker.io/apache/kie-kogito-db-migrator-tool:main"
     # SonataFlow base builder image used in the internal Dockerfile to build workflow applications in preview profile
     # Order of precedence is:
     # 1. SonataFlowPlatform in the given namespace
     # 2. This configuration
     # 3. The FROM in the Dockerfile in the operator's namespace "sonataflow-operator-builder-config" configMap.
     # If 1 or 2, the FROM tag will be replaced by the tag se there.
     # If empty the operator will use the default Apache Community one based on the current operator's version.
-    sonataFlowBaseBuilderImageTag: "docker.io/apache/incubator-kie-sonataflow-builder:main"
+    sonataFlowBaseBuilderImageTag: "docker.io/apache/incubator-kie-sonataflow-builder:1.36.0"
     # The image to use to deploy SonataFlow workflow images in devmode profile.
     # If empty the operator will use the default Apache Community one based on the current operator's version.
-    sonataFlowDevModeImageTag: "docker.io/apache/incubator-kie-sonataflow-devmode:main"
+    sonataFlowDevModeImageTag: "docker.io/apache/incubator-kie-sonataflow-devmode:1.36.0"
     # The default name of the builder configMap in the operator's namespace
     builderConfigMapName: "sonataflow-operator-builder-config"
     # Quarkus extensions required for workflows persistence. These extensions are used by the SonataFlow build system,
     # in cases where the workflow being built has configured postgresql persistence.
     postgreSQLPersistenceExtensions:
       - groupId: io.quarkus
         artifactId: quarkus-jdbc-postgresql
-        version: 3.8.6
       - groupId: io.quarkus
         artifactId: quarkus-agroal
-        version: 3.8.6
       - groupId: org.kie
         artifactId: kie-addons-quarkus-persistence-jdbc
-        version: 999-20241016-SNAPSHOT
     # If true, the workflow deployments will be configured to send accumulated workflow status change events to the Data
     # Index Service reducing the number of produced events. Set to false to send individual events.
     kogitoEventsGrouping: true
```

Comment:
![image](https://github.com/user-attachments/assets/d8200565-d853-4a8b-9b5c-d5075eb36c8d)
Above is for 1.35.0

Why this changes please? cc @ricardozanini

Reply:
We don't need the versions anymore since they are coming from the platform (aka the images itself have the KOGITO_VERSION and QUARKUS_VERSION already defined).